src/main/java/net/openhft/chronicle/threads/ExceptionHandlerStrategy.java

```diff
@@ -6,14 +6,19 @@
 import net.openhft.chronicle.core.threads.InvalidEventHandlerException;
 import net.openhft.chronicle.core.util.ObjectUtils;
 
+/**
+ * Strategy for how to deal with exceptions in handlers
+ */
 @FunctionalInterface
 public interface ExceptionHandlerStrategy {
+    @Deprecated(/* Remove in .25 */)
     String IMPL_PROPERTY = "el.exception.handler";
 
     /**
-     * TODO: use a builder pattern to construct EventLoops so we don't have this system property
+     * @deprecated Use {@link #EventGroup.builder()} - to be removed in .25
```

Comment:
as mentioned in the comments, maybe we just get rid of this altogether. I can't find any references to "el.exception.handler" anywhere in our code.